### PR TITLE
Move TLA+ models to tla subdirectory

### PR DESCRIPTION
**Pull request by ywelsch**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
-**/*.toolbox/*.pdf
-**/*.toolbox/*.tex
-**/*.pdf
+.DS_Store
+**/tla/*.toolbox/model
+**/tla/*.toolbox/*aux
+**/tla/*.toolbox/*.log
+**/tla/*.toolbox/*.pdf
+**/tla/*.toolbox/*.tex
+**/tla/*.pdf
```
Comment (DaveCTurner) [inline .gitignore]: Are these changes required? I thought `**/` meant "in any subdirectory"?

Comment (ywelsch) [reply .gitignore]: no, they're not required, the previous rules would have done fine. I thought that the previous ignore rules were maybe a bit overreaching (if we want to commit other stuff to this repo). With the above change, it's clear that those rules apply to all TLA models.
